dali/test/python/operator_1/test_numba_func_gpu.py

```diff
@@ -32,6 +32,9 @@ def check_env_compatibility():
         raise SkipTest()
     if cuda.runtime.get_version() > cuda.driver.driver.get_version():
         raise SkipTest()
+    if '.'.join(str(i) for i in cuda.driver.driver.get_version()) >= LooseVersion('12.0') and \
```

Comment:
Should the `'.'.join(str(i) for i in cuda.driver.driver.get_version())` be wrapped in LooseVersion as well? 
I am a bit wear about `str >= LooseVersion` comparison operator - if you swap arguments order it would be certain that LooseVersion's one is selected.

Reply:
Fixed